main/src/main/java/org/mobilitydata/gtfsvalidator/validator/FeedExpirationDateValidator.java

```diff
@@ -39,17 +40,17 @@
  */
 @GtfsValidator
 public class FeedExpirationDateValidator extends SingleEntityValidator<GtfsFeedInfo> {
-  private final ValidationContext context;
+  private final CurrentDateTime currentDateTime;
 
   @Inject
-  FeedExpirationDateValidator(ValidationContext context) {
-    this.context = context;
+  FeedExpirationDateValidator(ValidationContext validationContext) {
```

Comment:
Wait a sec! That should have CurrentDateTime as parameter.

Reply:
I am fine with merging this PR as-is and I can finish the injector part for you if you wish.

Reply:
Oops. Thanks for flagging that! Modifying.